webpack/assets/javascripts/react_app/components/LoginPage/LoginPageHelpers.js

```diff
@@ -0,0 +1,72 @@
+import React, { Fragment } from 'react';
+import { LoginPage } from 'patternfly-react';
+import { translate as __ } from '../../common/I18n';
+
+const { Alert } = LoginPage;
+
+export const adjustProps = ({
```

Comment:
I see this method redundant and I think this logic can be part of the `LoginPage` component.

Reply:
I am OK with it,
deleted `./helpers`,
moved this logic to `./LoginPage`